tools/speech_data_explorer/README.md

```diff
@@ -8,19 +8,37 @@ Features:
 - navigation across dataset (sorting, filtering)
 - inspection of individual utterances (waveform, spectrogram, audio player)
 - errors' analysis (Word Error Rate, Character Error Rate, Word Match Rate, Mean Word Accuracy, diff)
+- visually compare ASR models 
```

Comment:
- comparison of two ASR models using interactive word-level accuracy plot

Reply:
+